nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/LogFeature.java

```diff
@@ -71,6 +147,18 @@ public void setConsoleLogThreshold(String level) {
         hiddenAppender = newAppender;
     }
 
+    /**
+     * Sets a logger's level.
+     *
+     * @param loggerName the logger name to create or to update, it can be empty or null in this case the root logger's
+     *            will be taken
+     * @param level the leven to set
+     * @since 11.1
+     */
+    public void setLevel(String loggerName, String level) {
```

Comment:
Shouldn't we make it protected? because if one use it, does the logger level will be reset for next test?

Reply:
yep, this method  is unused, i added it for 
NXP-27269: Fix configuration variables names in StreamWorkManager and it forgot to remove it but now after the new devs it is unnecessary
thx